.github/workflows/build.yml

```diff
@@ -0,0 +1,58 @@
+name: Build Area Calculator
+
+on:
+  push:
+    paths-ignore:
+      - "**.md"
+      - "LICENSE"
+  pull_request:
+    paths-ignore:
+      - "**.md"
+      - "LICENSE"
+  workflow_dispatch:
+
+permissions:
+  contents: write
+
+
+jobs:
+  build:
+    runs-on: windows-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version-file: "pyproject.toml"
+          architecture: "x64"
+          cache: "pip"
+          
+      - name: Install dependencies
+        run: pip install . pyinstaller
+
+      - name: Build Executable
+        run: |
+          pyinstaller --onefile Area_Calculator.py
+
+      - name: Copy README
+        run: cp README.md ./dist/README.txt
+
+      - name: Upload Artifact
+        uses: actions/upload-artifact@v4
+        if: "!startsWith(github.ref, 'refs/tags/')"
```

Comment:
I think we can upload the artifact everytime, since we are running the pipeline on push anyway